sot_hpp/factory.py

```diff
@@ -9,21 +9,46 @@ class TaskFactory(ConstraintFactoryAbstract):
     def __init__ (self, graphfactory):
         super(TaskFactory, self).__init__ (graphfactory)
 
-    def buildGrasp (self, g, h):
+    def buildGrasp (self, g, h, otherGrasp=None):
         gf = self.graphfactory
         if h is None:
             return { 'gripper_open': EEPosture (gf.sotrobot, gf.gripperFrames [g], [0]) }
 
-        gripper_close  = EEPosture (gf.sotrobot, gf.gripperFrames [g], [-1])
+        gripper_close  = EEPosture (gf.sotrobot, gf.gripperFrames [g], [-0.2])
```

Comment:
A note for later: This is specific to the combination of Pyrene and the box. It should be externalized.